lib/Extension/Bundle/Installer/SettingsStoreAwareInstaller.php

```diff
@@ -83,6 +83,7 @@ protected function markInstalled(): void
 
                 if (!$executedMigrations->hasMigration($version)) {
                     $migrationResult = new ExecutionResult($version, Direction::UP);
+                    $this->dependencyFactory->getMetadataStorage()->ensureInitialized();
```

Comment:
Should we call $this->dependencyFactory->getMetadataStorage() only once and save this in a variable?